# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# 
# Translators:
# A Russo <quisquiliarum2001-habitica@yahoo.co.uk>, 2019
# Cesar Silva <cesarsilvabx@gmail.com>, 2017
# testsubject67 <deborinha97@hotmail.com>, 2014
# Eduardo Addad de Oliveira <eduardoaddad@hotmail.com>, 2018
# Francisco P. <sonhosdigitais@gmx.com>, 2013
# Henrique Rodrigues, 2017
# Koh Pyreit <inactive+kohpyreit@transifex.com>, 2013
# Manuela Silva <inactive+h_manuela_rodsilva@transifex.com>, 2015
# MS <manuelarodsilva@gmail.com>, 2014,2017-2018
# Manuela Silva <inactive+h_manuela_rodsilva@transifex.com>, 2015
# Pedro Albuquerque <pmra@gmx.com>, 2015
# Sérgio Marques <smarquespt@gmail.com>, 2016
# thor <thor_123@safe-mail.net>, 2015
# Tiago Filipe Rocha <rocha.t80@gmail.com>, 2014
# TiagoJMMC <tiagojmmc@gmail.com>, 2014
# Valter Fukuoka <dotsoft@gmail.com>, 2008
msgid ""
msgstr ""
"Project-Id-Version: Tor Project\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2019-04-13 08:42+0200\n"
"PO-Revision-Date: 2019-04-13 13:25+0000\n"
"Last-Translator: carolyn <carolyn@anhalt.org>\n"
"Language-Team: Portuguese (http://www.transifex.com/otf/torproject/language/pt/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: pt\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: ../tails_installer/creator.py:100
msgid "You must run this application as root"
msgstr "Deve executar esta aplicação como 'root'"

#: ../tails_installer/creator.py:146
msgid "Extracting live image to the target device..."
msgstr "A extrair a imagem live para o dispositivo de destino..."

#: ../tails_installer/creator.py:153
#, python-format
msgid "Wrote to device at %(speed)d MB/sec"
msgstr "Gravado no dispositivo a %(speed)d MB/seg."

#: ../tails_installer/creator.py:296
#, python-format
msgid ""
"There was a problem executing the following command: `%(command)s`.\n"
"A more detailed error log has been written to '%(filename)s'."
msgstr "Ocorreu um erro ao executar o seguinte comando: `%(command)s`.\nFoi guardado um registo mais detalhado do erro no '%(filename)s'."

#: ../tails_installer/creator.py:315
msgid "Verifying SHA1 checksum of LiveCD image..."
msgstr "A verificar a soma de verificação SHA1 da imagem do CD Live..."

#: ../tails_installer/creator.py:319
msgid "Verifying SHA256 checksum of LiveCD image..."
msgstr "A verificar a soma de verificação SHA256 da imagem do CD Live..."

#: ../tails_installer/creator.py:335
msgid ""
"Error: The SHA1 of your Live CD is invalid.  You can run this program with "
"the --noverify argument to bypass this verification check."
msgstr "Erro: A assinatura digital SHA1 do seu CD Live está incorreta. Pode executar este programa com o argumento --noverify para evitar esta verificação."

#: ../tails_installer/creator.py:341
msgid "Unknown ISO, skipping checksum verification"
msgstr "ISO desconhecido, a ignorar soma de verificação"

#: ../tails_installer/creator.py:353
#, python-format
msgid ""
"Not enough free space on device.\n"
"%dMB ISO + %dMB overlay > %dMB free space"
msgstr "Não existe espaço livre suficiente no dispositivo.\n%dMB ISO + %dMB overlay > %dMB espaço livre"

#: ../tails_installer/creator.py:360
#, python-format
msgid "Creating %sMB persistent overlay"
msgstr "A criar %sMB da área permanete ..."

#: ../tails_installer/creator.py:421
#, python-format
msgid "Unable to copy %(infile)s to %(outfile)s: %(message)s"
msgstr "Não foi possível copiar %(infile)s para %(outfile)s: %(message)s"

#: ../tails_installer/creator.py:435
msgid "Removing existing Live OS"
msgstr "A remover o SO Live existente"

#: ../tails_installer/creator.py:444 ../tails_installer/creator.py:457
#, python-format
msgid "Unable to chmod %(file)s: %(message)s"
msgstr "Não foi possível executar chmod %(file)s: %(message)s"

#: ../tails_installer/creator.py:450
#, python-format
msgid "Unable to remove file from previous LiveOS: %(message)s"
msgstr "Não é possível remover o ficheiro do LiveOS anterior: %(message)s"

#: ../tails_installer/creator.py:464
#, python-format
msgid "Unable to remove directory from previous LiveOS: %(message)s"
msgstr "Não é possível remover a diretoria do LiveOS anterior: %(message)s"

#: ../tails_installer/creator.py:512
#, python-format
msgid "Cannot find device %s"
msgstr "Não é possível encontrar %s"

#: ../tails_installer/creator.py:713
#, python-format
msgid "Unable to write on %(device)s, skipping."
msgstr "Não é possível gravar no %(device)s, a ignorar."

#: ../tails_installer/creator.py:743
#, python-format
msgid ""
"Some partitions of the target device %(device)s are mounted. They will be "
"unmounted before starting the installation process."
msgstr "Algumas partições do dispositivo %(device)s estão montadas. Elas serão desmontadas antes que o processo de instalação comece."

#: ../tails_installer/creator.py:786 ../tails_installer/creator.py:1010
msgid "Unknown filesystem.  Your device may need to be reformatted."
msgstr "Sistema de ficheiros desconhecido. Poderá ter que reformatar o seu dispositivo."

#: ../tails_installer/creator.py:789 ../tails_installer/creator.py:1013
#, python-format
msgid "Unsupported filesystem: %s"
msgstr "Sistema de ficheiros não suportado: %s"

#: ../tails_installer/creator.py:807
#, python-format
msgid "Unknown GLib exception while trying to mount device: %(message)s"
msgstr "Exceção GLib desconhecida enquanto tentava montar o dispositivo: %(message)s"

#: ../tails_installer/creator.py:812
#, python-format
msgid "Unable to mount device: %(message)s"
msgstr "Não é possível montar o dispositivo: %(message)s"

#: ../tails_installer/creator.py:817
msgid "No mount points found"
msgstr "Não foram encontrados pontos de montagem"

#: ../tails_installer/creator.py:828
#, python-format
msgid "Entering unmount_device for '%(device)s'"
msgstr "A entrar em unmount_device para '%(device)s'"

#: ../tails_installer/creator.py:838
#, python-format
msgid "Unmounting mounted filesystems on '%(device)s'"
msgstr "A desmontar o sistema de ficheiros montados no '%(device)s'"

#: ../tails_installer/creator.py:842
#, python-format
msgid "Unmounting '%(udi)s' on '%(device)s'"
msgstr "A desmontar '%(udi)s' no '%(device)s'"

#: ../tails_installer/creator.py:853
#, python-format
msgid "Mount %s exists after unmounting"
msgstr "O ponto de montagem %s existe depois de desmontado"

#: ../tails_installer/creator.py:866
#, python-format
msgid "Partitioning device %(device)s"
msgstr "A particionar o dispositivo %(device)s"

#: ../tails_installer/creator.py:995
#, python-format
msgid "Unsupported device '%(device)s', please report a bug."
msgstr "Dispositivo '%(device)s' não suportado, por favor, comunique um bug."

#: ../tails_installer/creator.py:998
msgid "Trying to continue anyway."
msgstr "A tentar continuar de qualquer forma."

#: ../tails_installer/creator.py:1007 ../tails_installer/creator.py:1405
msgid "Verifying filesystem..."
msgstr "A verificar o sistema de ficheiros ..."

#: ../tails_installer/creator.py:1031
#, python-format
msgid "Unable to change volume label: %(message)s"
msgstr "Não é possível alterar o nome do volume: %(message)s"

#: ../tails_installer/creator.py:1037 ../tails_installer/creator.py:1440
msgid "Installing bootloader..."
msgstr "A instalar o carregador de arranque..."

#: ../tails_installer/creator.py:1064
#, python-format
msgid "Could not find the '%s' COM32 module"
msgstr "Impossível encontrar %s modulo COM32"

#: ../tails_installer/creator.py:1072 ../tails_installer/creator.py:1458
#, python-format
msgid "Removing %(file)s"
msgstr "A remover %(file)s"

#: ../tails_installer/creator.py:1186
#, python-format
msgid "%s already bootable"
msgstr "%s já é de arranque"

#: ../tails_installer/creator.py:1206
msgid "Unable to find partition"
msgstr "Não foi possível encontrar a partição"

#: ../tails_installer/creator.py:1229
#, python-format
msgid "Formatting %(device)s as FAT32"
msgstr "A formatar o %(device)s como FAT32"

#: ../tails_installer/creator.py:1289
msgid "Could not find syslinux' gptmbr.bin"
msgstr "Impossível encontrar syslinux' gptmbr.bin"

#: ../tails_installer/creator.py:1302
#, python-format
msgid "Reading extracted MBR from %s"
msgstr "A ler MBR extraído de %s"

#: ../tails_installer/creator.py:1306
#, python-format
msgid "Could not read the extracted MBR from %(path)s"
msgstr "Não foi possível ler MBR extraído de %(path)s"

#: ../tails_installer/creator.py:1319 ../tails_installer/creator.py:1320
#, python-format
msgid "Resetting Master Boot Record of %s"
msgstr "A reiniciar o 'Master Boot Record' de %s"

#: ../tails_installer/creator.py:1325
msgid "Drive is a loopback, skipping MBR reset"
msgstr "A unidade é um 'loopback', a ignorar o reinício do MBR"

#: ../tails_installer/creator.py:1329 ../tails_installer/creator.py:1589
#, python-format
msgid "Calculating the SHA1 of %s"
msgstr "A calcular o SHA1 de %s"

#: ../tails_installer/creator.py:1354
msgid "Synchronizing data on disk..."
msgstr "A sincronizar os dados no disco..."

#: ../tails_installer/creator.py:1397
msgid "Error probing device"
msgstr "Erro ao examinar o dispositivo"

#: ../tails_installer/creator.py:1399
msgid "Unable to find any supported device"
msgstr "Não foi possível encontrar um dispositivo suportado"

#: ../tails_installer/creator.py:1409
msgid ""
"Make sure your USB key is plugged in and formatted with the FAT filesystem"
msgstr "Certifique-se que a pen USB está ligada e formatada com o sistema de ficheiros FAT"

#: ../tails_installer/creator.py:1412
#, python-format
msgid ""
"Unsupported filesystem: %s\n"
"Please backup and format your USB key with the FAT filesystem."
msgstr "Sistema de ficheiros não suportado: %s\nPor favor, efetue uma cópia de segurança e formate a sua pen USB com o sistema de ficheiros FAT."

#: ../tails_installer/creator.py:1481
msgid ""
"Unable to get Win32_LogicalDisk; win32com query did not return any results"
msgstr "Não foi possível obter o Win32_LogicalDisk; a consulta win32com não devolveu quaisquer resultados"

#: ../tails_installer/creator.py:1536
msgid "Cannot find"
msgstr "Não é possível encontrar"

#: ../tails_installer/creator.py:1537
msgid ""
"Make sure to extract the entire tails-installer zip file before running this"
" program."
msgstr "Certifique-se que extrai todo o ficheiro zip do instalador tails antes de executar este programa."

#: ../tails_installer/gui.py:69
#, python-format
msgid "Unknown release: %s"
msgstr "Lançamento desconhecido: %s"

#: ../tails_installer/gui.py:73
#, python-format
msgid "Downloading %s..."
msgstr "A transferir %s..."

#: ../tails_installer/gui.py:213
msgid ""
"Error: Cannot set the label or obtain the UUID of your device.  Unable to "
"continue."
msgstr "Erro: Não é possível definir o nome ou obter o UUID do seu dispositivo. Não é possível continuar."

#: ../tails_installer/gui.py:260
#, python-format
msgid "Installation complete! (%s)"
msgstr "Instalação completa! (%s)"

#: ../tails_installer/gui.py:265
msgid "Tails installation failed!"
msgstr "A instalação de Tails falhou!"

#: ../tails_installer/gui.py:369
msgid ""
"Warning: This tool needs to be run as an Administrator. To do this, right "
"click on the icon and open the Properties. Under the Compatibility tab, "
"check the \"Run this program as an administrator\" box."
msgstr "Aviso: esta ferramenta precisa de ser executada como um Administrador. Para o fazer, clique direito no ícone e abra as Propriedades. No separador 'Compatibilidade', selecione \"Executar como Administrador\"."

#: ../tails_installer/gui.py:381
msgid "Tails Installer"
msgstr "Ficheiro de instalação Tails"

#: ../tails_installer/gui.py:441
msgid "Tails Installer is deprecated in Debian"
msgstr ""

#: ../tails_installer/gui.py:443
msgid ""
"To install Tails from scratch, use GNOME Disks instead.\n"
"<a href='https://tails.boum.org/install/linux/usb-overview'>See the installation instructions</a>\n"
"\n"
"To upgrade Tails, do an automatic upgrade from Tails or a manual upgrade from Debian using a second USB stick.\n"
"<a href='https://tails.boum.org/upgrade/tails-overview'>See the manual upgrade instructions</a>"
msgstr ""

#: ../tails_installer/gui.py:450 ../data/tails-installer.ui.h:2
msgid "Clone the current Tails"
msgstr "Clone oTails atual"

#: ../tails_installer/gui.py:457 ../data/tails-installer.ui.h:3
msgid "Use a downloaded Tails ISO image"
msgstr "Utilize a imagem Tails ISO transferida"

#: ../tails_installer/gui.py:494 ../tails_installer/gui.py:815
msgid "Upgrade"
msgstr "Atualizar"

#: ../tails_installer/gui.py:496
msgid "Manual Upgrade Instructions"
msgstr "Instruções de Atualização Manual"

#: ../tails_installer/gui.py:498
msgid "https://tails.boum.org/upgrade/"
msgstr "https://tails.boum.org/upgrade/"

#: ../tails_installer/gui.py:506 ../tails_installer/gui.py:727
#: ../tails_installer/gui.py:792 ../data/tails-installer.ui.h:7
msgid "Install"
msgstr "Instalar"

#: ../tails_installer/gui.py:509 ../data/tails-installer.ui.h:1
msgid "Installation Instructions"
msgstr "Guias de instalação"

#: ../tails_installer/gui.py:511
msgid "https://tails.boum.org/install/"
msgstr "https://tails.boum.org/install/"

#: ../tails_installer/gui.py:517
#, python-format
msgid "%(size)s %(vendor)s %(model)s device (%(device)s)"
msgstr "%(size)s %(vendor)s %(model)s dispositivo (%(device)s)"

#: ../tails_installer/gui.py:529
msgid "No ISO image selected"
msgstr "Nenhuma imagem ISO foi selecionada"

#: ../tails_installer/gui.py:530
msgid "Please select a Tails ISO image."
msgstr "Por favor selecione o ficheiro de imagem Tails ISO"

#: ../tails_installer/gui.py:572
msgid "No device suitable to install Tails could be found"
msgstr "Não foi encontrado nenhum dispositivo adequado para a instalação de Tails"

#: ../tails_installer/gui.py:574
#, python-format
msgid "Please plug a USB flash drive or SD card of at least %0.1f GB."
msgstr "Por favor, insira uma PEN USB ou cartão SD com pelo menos %0.1f GB."

#: ../tails_installer/gui.py:608
#, python-format
msgid ""
"The USB stick \"%(pretty_name)s\" is configured as non-removable by its "
"manufacturer and Tails will fail to start from it. Please try installing on "
"a different model."
msgstr ""

#: ../tails_installer/gui.py:618
#, python-format
msgid ""
"The device \"%(pretty_name)s\" is too small to install Tails (at least "
"%(size)s GB is required)."
msgstr "O dispositivo \"%(pretty_name)s\" é muito pequeno para instalar o Tails (é necessário pelo menos %(size)s GB)."

#: ../tails_installer/gui.py:631
#, python-format
msgid ""
"To upgrade device \"%(pretty_name)s\" from this Tails, you need to use a downloaded Tails ISO image:\n"
"https://tails.boum.org/install/download"
msgstr "Para atualizar o dispositivo %(pretty_name)s deste Tails, você precisa usar uma imagem ISO do Tails baixada:\nhttps://tails.boum.org/install/download"

#: ../tails_installer/gui.py:652
msgid "An error happened while installing Tails"
msgstr "Aconteceu um erro durante a instalação de Tails"

#: ../tails_installer/gui.py:664
msgid "Refreshing releases..."
msgstr "A atualizar os lançamentos..."

#: ../tails_installer/gui.py:669
msgid "Releases updated!"
msgstr "Lançamentos atualizados!"

#: ../tails_installer/gui.py:722
msgid "Installation complete!"
msgstr "Instalação completa!"

#: ../tails_installer/gui.py:738
msgid "Cancel"
msgstr "Cancelar"

#: ../tails_installer/gui.py:774
msgid "Unable to mount device"
msgstr "Não é possível montar o dispositivo"

#: ../tails_installer/gui.py:781 ../tails_installer/gui.py:814
msgid "Confirm the target USB stick"
msgstr "Confirme o dispositivo USB alvo"

#: ../tails_installer/gui.py:782
#, python-format
msgid ""
"%(size)s %(vendor)s %(model)s device (%(device)s)\n"
"\n"
"All data on this USB stick will be lost."
msgstr "%(size)s %(vendor)s %(model)s dispositivo (%(device)s)\n\nTodos os dados na pen USB serão perdidos."

#: ../tails_installer/gui.py:801
#, python-format
msgid "%(parent_size)s %(vendor)s %(model)s device (%(device)s)"
msgstr "%(parent_size)s %(vendor)s %(model)s dispositivo (%(device)s)"

#: ../tails_installer/gui.py:809
msgid ""
"\n"
"\n"
"The persistent storage on this USB stick will be preserved."
msgstr "\n\nO armazenamento persistente nesta pen USB será preservado."

#: ../tails_installer/gui.py:810
#, python-format
msgid "%(description)s%(persistence_message)s"
msgstr "%(description)s%(persistence_message)s"

#: ../tails_installer/gui.py:853
msgid "Download complete!"
msgstr "Transferência completa!"

#: ../tails_installer/gui.py:857
msgid "Download failed: "
msgstr "A transferência falhou:"

#: ../tails_installer/gui.py:858
msgid "You can try again to resume your download"
msgstr "Pode tentar novamente para retomar a sua transferência"

#: ../tails_installer/gui.py:866
msgid ""
"The selected file is unreadable. Please fix its permissions or select "
"another file."
msgstr "O ficheiro selecionado não é legível. Por favor corrija as permissões ou selecione outro ficheiro."

#: ../tails_installer/gui.py:872
msgid ""
"Unable to use the selected file.  You may have better luck if you move your "
"ISO to the root of your drive (ie: C:\\)"
msgstr "Não é possível utilizar o ficheiro selecionado. É capaz de ter mais sorte se mover o seu ficheiro ISO para a raiz da sua unidade (ex.: C:\\)."

#: ../tails_installer/gui.py:878
#, python-format
msgid "%(filename)s selected"
msgstr "%(filename)s selecionado"

#: ../tails_installer/source.py:29
msgid "Unable to find LiveOS on ISO"
msgstr "Não foi possível encontrar o LiveOS na imagem ISO"

#: ../tails_installer/source.py:35
#, python-format
msgid "Could not guess underlying block device: %s"
msgstr "Não foi possível adivinhar o dispositivo de bloco subjacente: %s"

#: ../tails_installer/source.py:50
#, python-format
msgid ""
"There was a problem executing `%s`.\n"
"%s\n"
"%s"
msgstr "Ocorreu um problema ao executar `%s`.\n%s\n%s"

#: ../tails_installer/source.py:64
#, python-format
msgid "'%s' does not exist"
msgstr "'%s' não existe"

#: ../tails_installer/source.py:66
#, python-format
msgid "'%s' is not a directory"
msgstr "%sNão é um diretório"

#: ../tails_installer/source.py:76
#, python-format
msgid "Skipping '%(filename)s'"
msgstr "A ignorar '%(filename)s'"

#: ../tails_installer/utils.py:55
#, python-format
msgid ""
"There was a problem executing `%s`.%s\n"
"%s"
msgstr "Ocorreu um problema ao executar `%s`.%s\n%s"

#: ../tails_installer/utils.py:135
msgid "Could not open device for writing."
msgstr "Não foi possível escrever no dispositivo."

#: ../data/tails-installer.ui.h:4
msgid "Select a distribution to download:"
msgstr "Selecione uma versão para transferência:"

#: ../data/tails-installer.ui.h:5
msgid "Target USB stick:"
msgstr "Dispositivo USB de destino:"

#: ../data/tails-installer.ui.h:6
msgid "Reinstall (delete all data)"
msgstr "Reinstalar (apagar tudo)"
